### DISCUSSION TITLE
HTML  textobject support  and emmet-ls not working

### DISCUSSION THREAD

**Discussion by Aritra640** [helix-editor/helix#11862] [Troubleshooting] [answered]
Whenever I try to use emmet-ls as a lsp for html it shows language server exited, also there s no text object support for html 
here is my language.toml setup
`
[[language]]
name = "go"
auto-format = true
formatter = { command = "goimports" }

[[language]]
name = "cpp"
formatter = {command = "clang-format"}

 [language-server.emmet-ls]
 command = "emmet-ls"
 args = ["--studio"]

[[language]]
name = "html"
formatter = {command = "prettier"}
language-servers = ["vscode-html-language-server","emmet-ls"]
auto-format = true

 [language-server.vscode-css-language-server.config]
 css = { validate = { enable = true } }
 scss = { validate = { enable = true } }
 less = { validate = { enable = true } }
 provideFormatter = true

[[language]]
 name = "css"
 formatter = { command = 'npx', args = ["prettier", "--parser", "css"] }
 language-servers = [ "vscode-css-language-server", "tailwindcss-react", "tailwindcss-ls"]
 auto-format = true

[[language]]
name = "javascript"
auto-format = true
formatter = {command = "prettier"}

[[language]]
name = "python"
auto-format = true
formatter = {command = "isort"}

[[language]]
name = "rust"
auto-format = true
formatter = {command = "rustfmt"}

`

**Comment by David-Else** [comment]
Text objects for HTML are not merged yet: https://github.com/helix-editor/helix/pull/11158

**Comment by wadaerna** [comment, marked as answer]
Hi, I believe the issue is that you have `args = ["--studio"]` and it should be `args = ["--stdio"]`

**Reply by Aritra640** [reply]
Thanks it worked
